src/test/java/org/xstefank/model/PersistentListTest.java

```diff
@@ -0,0 +1,37 @@
+package org.xstefank.model;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.xstefank.TestUtils;
+
+public class PersistentListTest {
+
+    private final String USERNAME = "testUser";
+    private final String FILE_NAME = "testList.txt";
+
+    private PersistentList testUserList;
+
+    @Before
+    public void before() {
+        testUserList = new PersistentList(TestUtils.TARGET_DIR, FILE_NAME);
```

Comment:
creating in testList.txt in "basedir/target" but after is looking for it to delete it in "src/test/resources" ?